packages/bundle/src/renderMarkdown.js

```diff
@@ -65,6 +69,14 @@ const customMarkdownIt = new MarkdownIt({
       tokens[index].attrs[targetAttrIndex][1] = '_blank';
     } else {
       tokens[index].attrPush(['target', '_blank']);
+      tokens[index].attrPush(['class', 'externalLink']);
+    }
+
+    const urlAttrIndex = tokens[index].attrIndex('href');
+    if (isExternal(urlAttrIndex)) {
```

Comment:
These should be conditionally added since the markdown author could supply their own title

Reply:
Can you elaborate on the condition part? In the commit 6039f009617bdc366c1085f9938dce433b977236 I'm adding localized strings as also requested by @compulim